releasenotes/source/_deconst.json

```diff
@@ -1,5 +1,5 @@
 {
-    "contentIDBase": "https://github.com/rcbops/rpc-openstack/v13",
+    "contentIDBase": "https://github.com/rcbops/rpc-openstack/master/",
```

Comment:
This branch is mitaka-13.0, does it matter that master is being referenced in multiple places?